packages/manager/apps/web/client/app/dns-zone/history/history.routing.js

```diff
@@ -0,0 +1,59 @@
+import controllerDiff from './dns-zone-diff-tool-viewer/history.controller';
+import templateDiff from './dns-zone-diff-tool-viewer/history.html';
+import controllerDashboard from './dashboard/dashboard.controller';
+import templateDashboard from './dashboard/dashboard.html';
```

Comment:
```suggestion
import zoneDiffToolViewerController from './dns-zone-diff-tool-viewer/history.controller';
import zoneDiffToolViewerTemplate from './dns-zone-diff-tool-viewer/history.html';
import dashboardController from './dashboard/dashboard.controller';
import dashboardTemplate from './dashboard/dashboard.html';
```

Reply:
Using components instead to simplify imports here